usage/dist/licensing/overrides.yaml

```diff
@@ -176,6 +200,15 @@
   notice: Copyright (c) Jeremy Ashkenas, DocumentCloud Inc (2009-2013)
   license: MIT
 
+# used in CLI (and in docs)
+- id: underscore.js:1.7.0
+  version: 1.7.0
+  files: underscore*.{js,map}
+  url: http://underscorejs.org
+  organization: { name: "DocumentCloud Inc", url: "http://www.documentcloud.org/" }
+  notice: Copyright (c) Jeremy Ashkenas, DocumentCloud Inc (2009-2014)
```

Comment:
(c) 2009-2014 Jeremy Ashkenas, DocumentCloud and Investigative Reporters & Editors

See [underscore-min.js](https://github.com/apache/incubator-brooklyn/blob/master/usage/cli/src/main/resources/brooklyn/item-lister/statics/style/js/underscore-min.js)